configure

```diff
@@ -425,6 +425,38 @@ sub run_command {
   );
 }
 
+sub git_clone {
+  my $dest = shift;
+  my $url = shift;
+  my $branch = shift;
+  print "Cloning git repo $url branch $branch\n";
+  my $rv = run_command(['git', 'clone', '-q', '--depth=1', '-b', $branch, $url, $dest]);
+  if (!$rv) {
+    run_command(['git', '--no-pager', 'log', '-1', '--oneline'], chdir => $dest);
+  }
+  return $rv;
+}
+
+sub git_sm_prop {
```

Comment:
```suggestion
sub git_submodule_prop {
```

Or change the error message on 445, but I think the longer function name is more clear